modules/ROOT/pages/hdfs/hdfs-connector.adoc

```diff
@@ -1,157 +1,64 @@
 
-Release Notes: xref:release-notes::connector/hdfs-connector-release-notes-mule-4.adoc[HDFS Connector Release Notes] +
-Exchange: https://www.mulesoft.com/exchange/org.mule.connectors/mule-hdfs-connector/[Hadoop (HDFS) Connector]
+For information about compatibility and fixed issues, see xref:release-notes::connector/hdfs-connector-release-notes-mule-4.adoc[Hadoop (HDFS) Connector Release Notes] 
 
 == Prerequisites
 
-To use this information, you should be familiar with HDFS, Mule, Anypoint Platform, and Anypoint Studio.
+To use this connector, you must be familiar with the following concepts:
 
-To use the HDFS connector, you need:
+* Anypoint connectors
+* Mule runtime engine (Mule)
+* Elements in a Mule flow
+* Global elements
+* How to create a Mule app using Design Center or Anypoint Studio
 
-* Anypoint Studio version 7.0 or above - An instance of Anypoint Studio. If you do not use Anypoint Studio for development.
-* An instance of Hadoop Distributed File System  up and running.
+Before creating an app, you must have:
 
-== Compatibility
+* Access to the Hadoop (HDFS) target resource and Anypoint Platform 
+* Anypoint Studio version 7.0 or later 
+* An instance of Hadoop Distributed File System running
 
-HDFS Connector is compatible with the following:
 
-[%header,width="100%",cols="50%,50%"]
-|===
-|Application/Service|Version
-|Mule Runtime |4.0 or newer
-|Apache Hadoop |2.8.1 or newer
-|===
+== Audience
 
-== POM File Information
+* Starting user
++
+To create your Mule app, see:
+** xref:hdfs/hdfs-connector-studio.adoc[Studio Configuration]
+** xref:hdfs/hdfs-connector-examples.adoc[Examples], which has use cases for the connector
++
+* Power user
++
+Read the following topics: 
+** xref:hdfs/hdfs-connector-xml-maven.adoc[XML and Maven Support]
+** xref:hdfs/hdfs-connector-examples.adoc[Examples]  
 
-If you are coding your connector in XML, set the pom.xml file as follows:
+== Connection Types
 
-[source,xml,linenums]
-----
-<dependency>
-  <groupId>org.mule.connectors</groupId>
-  <artifactId>mule-hdfs-connector</artifactId>
-  <version>RELEASE</version>
-  <classifier>mule-plugin</classifier>
-</dependency>
-----
+The following connection authentication types are supported:
 
-Mule converts RELEASE to the latest version. To specify a version, view 
-Anypoint Exchange and click *Dependency Snippets*.
+* Kerberos +
+This connection type uses the Kerberos principal as the username, and the keytab path you specify is used to obtain a ticket granting ticket (TGT) from the authorization server. 
+* Simple +
+ For this connection type, you provide the name node of the file system to which to connect, and it is passed to the HDFS client as the default name key.
```

Comment:
parallelism
```suggestion
This connection type passes the name of the node that contains the file system you want to connect to as the default name key.
```
Hmm...reread to see if this still makes sense. Might need a tweak.